static/src/stylesheets/module/content/_media-player.scss

```diff
@@ -20,9 +20,13 @@ $ima-controls-height: 70px;
     -webkit-transform: translateZ(0); // fixes iOS hover bug
 }
 .gu-media-wrapper--audio {
-    background-color: $vjs-control-colour;
+    background-color: rgba($vjs-control-colour, .8);
```

Comment:
Again- should ew be adding things like this to the palette so as not to have someone add a `.75` version down the line?